packages/cli/src/cmds/assert/types.d.ts

```diff
@@ -1 +1,7 @@
 export type Scope = 'appmap' | 'http_server_request' | 'sql_query' | 'function';
+
+export interface AssertionFailure {
+  appMapName: string;
+  event: object;
```

Comment:
This should be `any` to prevent you needing to `ts-ignore` any time you access the properties of `event`.